tests/alerts/plugins/test_port_scan_enrichment.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+import os
+import sys
+
+plugin_path = os.path.join(os.path.dirname(__file__), '../../../alerts/plugins')
+sys.path.append(plugin_path)
+
+from port_scan_enrichment import enrich
+
+
+def mock_search_fn(results):
+    def search_fn(_query):
+        return results
+
+    return search_fn
+
+
+class TestPortScanEnrichment(object):
+    def test_alert_enriched(self):
+        results = {
+            'hits': [
+                {
+                    '_source': {
+                        'details': {
+                            'destinationipaddress': '1.2.3.4',
+                            'destinationport': 80
+                        },
+                        'timestamp': '30 minutes ago'
```

Comment:
Since the timestamp will be ISO 8601 formatted, can we have the sample data be the same? IE: `2016-07-13 22:33:31.625443+00:00`